src/main/java/com/redhat/exhort/integration/providers/osvnvd/OsvNvdIntegration.java

```diff
@@ -60,6 +61,21 @@ public void configure() throws Exception {
       .to(vertxHttp("{{api.osvnvd.host}}"))
       .transform().method(responseHandler, "responseToIssues");
 
+    from(direct("osvNvdHealthCheck"))
+      .routeId("osvNvdHealthCheck")
+      .setProperty(Constants.PROVIDER_NAME, constant(Constants.OSV_NVD_PROVIDER))
+      .process(this::processHealthRequest)
+      .circuitBreaker()
+        .faultToleranceConfiguration()
+          .timeoutEnabled(true)
+          .timeoutDuration(timeout)
+        .end()
+        .to(vertxHttp("{{api.osvnvd.management.host}}"))
+      .onFallback()
+        .setBody(ExpressionBuilder.bodyExpression("osvNvd Service is down"))
```

Comment:
I think here you wanted to use `constant` this is why the pipeline was failing with error `Exception occurred during execution on the exchange: Exchange[A36E02E84B42677-0000000000000002]`
```suggestion
        .setBody(constant(Constants.OSV_NVD_PROVIDER + " Service is down"))
```

Reply:
You're right, i meant constant, my bad.
Fixed.